### DISCUSSION TITLE
missing inline.rs

### DISCUSSION THREAD

**Discussion by smartekIT** [rustdesk/rustdesk#1284] [Q&A]
Hi,
When building in windows for windows client using <pre>python3 build.py -f with_rc</pre>

first it doesn't recognise (with_rc) so it goes by (inline)
now it's complaining no inline.rs in <pre>src\ui\inline.rs</pre>

Thanks in advance for your help

**Comment by ghost** [comment]
Hello man, 
I encountered the same problem as you did and I've found some help on the rustdesk's discord. 
According to your problem, you must have installed python from official Python website. 
If you did so, you need to modify the "build.py" script.

On line **231**, you must modify the command like this:
- actual command: os.system('python3 res/inline-sciter.py')
- modified command: os.system('py res/inline-sciter.py')

It's because python3 is not a recognized command in windows unless you download python from Microsoft store.

Regarding you're RC question I can't help you.

Hope that'll help you even though you're question is 2 month old.

**Reply by smartekIT** [reply]
Thank you so much, i really appreciate your help as i asked this long time ago.
I tried your solution but it still the same, getting: unrecognized features 'with_rc'
